tests/drivers/can/host/testcase.yaml

```diff
@@ -0,0 +1,17 @@
+common:
+  tags:
+    - drivers
+    - can
+  depends_on: can
```

Comment:
note: this seems redundant as we filter by dt_chosen_enabled("zephyr,canbus"), which implies can

Reply:
It's not - `depends_on` filters at test suite collection time (filtering out a lot of boards), whereas the `chosen` filter needs to process the DTS. The latter is mostly a precaution, since all boards supporting CAN should have a `zephyr,canbus` chosen.